tests/libres_tests/res/enkf/test_runpath_list_dump.py

```diff
@@ -39,6 +39,9 @@ def _verify_runpath_rendering(self, itr):
             res.runpaths._runpath_format = runpath_fmt
             res.runpaths._job_name_format = jobname_fmt
 
+            for i in range(num_realizations):
+                res.set_geo_id(str(10 * i), i, itr)
```

Comment:
We should probably make sure that this works with batchsimulator.

Reply:
Quite sure it works now, not sure why it worked before, as it never replaced `<GEO_ID>`, so that was a part of the `run_path`